frontend/src/app.jsx

```diff
@@ -1,58 +1,57 @@
 import { useEffect, useState } from "react";
-
+import Task from "./components/task.jsx";
```

Comment:
We were missing the import of the `<Task />` component before using it in line 51.